### ISSUE TITLE
v2.11 was built with go 1.10 instead of 1.11

### ISSUE THREAD

**Issue by helgi** [helm/helm#4696]
https://circleci.com/gh/helm/helm/6454 - see the env output or the configuration tab

https://github.com/helm/helm/pull/4637 has that change - Did it arrive too late in the release window? 

Was hoping to have 1.11 and Sprig 2.16 (landed later) together. 1.11 added a nice thing for templates (https://golang.org/doc/go1.11 - `Modifying template variables via assignments is now permitted via the = token:`)

**Comment by bacongobbler** [comment]
Yes, sadly that missed the window. It'll be in 2.12 though!

**Comment by bacongobbler** [comment]
To explain why go 1.11 wasn't merged in and why sprig was:

Both of these PRs were merged after we cut the first release candidate. When evaluating which PRs should be squeezed in and re-cut another RC, we found that sprig 2.16 contained a few substantial bug fixes, so we considered it for merging into the release branch. However, compiling Helm against a newer version of Go has bitten us in the past, and since none of us felt comfortable with Go 1.11 to say whether or not it'd introduce breaking changes, we decided to hold off until the next release.

Hope that helps!
